README.md

```diff
@@ -502,6 +505,24 @@ Flags:
   -u, --username    Username (default: <none>)
 ```
 
+### Logout from a Registry
+
+```console
+$ img logout -h
+Usage: img logout [SERVER]
+
+Log out from a Docker registry.
+
+If no server is specified, the default (https://index.docker.io/v1/) is used.
+
+Flags:
+
+  -b, --backend  backend for snapshots ([auto native overlayfs]) (default: auto)
+  -d, --debug    enable debug logging (default: false)
+  -s, --state    directory to hold the global state (default: /home/weston/.local/share/img)
```

Comment:
hmm, is this _really_ the default? ;)

Reply:
i mean the user doesn't know who `weston` is.

Reply:
good eye